.github/workflows/default.yml

```diff
@@ -1,5 +1,8 @@
 name: Default
 
+env:
```

Comment:
Setup go hat auch die Möglichkeit die Version aus der `go.mod` auszulesen: https://github.com/actions/setup-go#getting-go-version-from-the-gomod-file

Reply:
Sehr cool- ich würde trotzdem lieber mit der letzten Version bauen anstatt mit der genau angegebenen.

Reply:
Nach meinem Textverständnis macht das genau das

> As the go.mod file contains only major and minor (e.g. 1.18) tags, the action will search for the latest available patch version sequentially in the runner's directory with the cached tools

Reply:
Ja, es nimmt aber kein 1.19.